storage_service/common/management/commands/create_user.py

```diff
@@ -0,0 +1,44 @@
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand, CommandError
+
+from tastypie.models import ApiKey
+
+
+class Command(BaseCommand):
+    help = 'Create user with all the arguments specified. ' \
+        'If the user already exists its attributes will be updated.'
+    args = ('username', 'email', 'password', 'api_key')
+
+    def add_arguments(self, parser):
+        parser.add_argument('--username', required=True)
+        parser.add_argument('--email', required=True)
+        parser.add_argument('--password', required=True)
+        parser.add_argument('--api-key', required=True)
+
+    def handle(self, *args, **options):
+        UserModel = get_user_model()
+        user = None
+        try:
+            user = UserModel._default_manager.using('default').get(**{UserModel.USERNAME_FIELD: options['username']})
+            print('User found.')
+        except UserModel.DoesNotExist:
+            user = UserModel._default_manager.using('default').create_superuser(options['username'], options['email'], options['password'])
+            print('User could not be found, one was created.')
+
+        if user is None:
+            raise CommandError('User not found!')
+
+        user.email = options['email']
+        user.set_password(options['password'])
+        user.save()
+        print('User updated.')
+
+        try:
```

Comment:
I think that would be (I've not tested this)
```
ApiKey.objects.create_or_update(user=user, defaults={'key': options['api_key']})
```

and if you want to know whether it was created or updated:
```
obj, created = ApiKey.objects.create_or_update(user=user, defaults={'key': options['api_key']})
if created:
    print('API key created')
```

etc.